.binder/requirements.txt

```diff
@@ -5,4 +5,5 @@ scikit-image
 pandas
 sphinx-gallery
 scikit-learn
+shap
```

Comment:
How stable are the APIs of shap that we rely on here? should we pin this dependency?

Reply:
Small follow-up: do you know if APIs of shap are stable enough, @lucyleeow?

Reply:
It should be